bear.profile

```diff
@@ -45,4 +48,14 @@ function bear_form_node_form_alter(&$form, FormStateInterface $form_state) {
   if (isset($form['sticky'])) {
     $form['sticky']['#access'] = FALSE;
   }
+
+  $user = \Drupal::currentUser();
+  if (\Drupal::moduleHandler()->moduleExists('workbench_moderation') && isset($form['moderation_state'])) {
```

Comment:
I dont even think workbench moderation has a d8 version, so you dont need to check for it